deltachat-ios.xcodeproj/project.pbxproj

```diff
@@ -24,6 +25,8 @@
 		302B84C6239676F0001C261F /* AvatarHelper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 30AC265E237F1807002A943F /* AvatarHelper.swift */; };
 		302B84C72396770B001C261F /* RelayHelper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 302B84C42396627F001C261F /* RelayHelper.swift */; };
 		302B84CE2397F6CD001C261F /* URL+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 302B84CD2397F6CD001C261F /* URL+Extension.swift */; };
+		302D5450268B6B2300A8B271 /* MessageUtils.swift in Sources */ = {isa = PBXBuildFile; fileRef = 302D544F268B6B2300A8B271 /* MessageUtils.swift */; };
+		302D5454268B84CB00A8B271 /* SettingsVideoChatInstanceController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 302D5453268B84CB00A8B271 /* SettingsVideoChatInstanceController.swift */; };
```

Comment:
just a short comment, although still WIP: 

`SettingsVideoChatInstanceController.swift `  is a `ViewController`, right?

so we should add that to the name; even if some files are not named that way, we should try to get some consistency over time.

as other settings view controller start with settings, maybe `SettingsVideoChatViewController.swift`? that would also be fine if some future settings won't affect the instance only - and is even a bit shorter.